Cargo.toml

```diff
@@ -21,7 +21,7 @@ cid = { default-features = false, version = "0.5" }
 dirs = { default-features = false, version = "3.0" }
 domain = { default-features = false, features = ["resolv"], git = "https://github.com/nlnetlabs/domain", rev="084964" }
 futures = { default-features = false, features = ["compat", "io-compat"], version = "0.3.5" }
-ipfs-unixfs = { path = "unixfs" }
+ipfs-unixfs = { features = ["std", "with-filetime"], path = "unixfs" }
```

Comment:
Leaving `with-filetime` or `filetime` out would allow you to shave at least one optional dependency off; I don't think this is used from anywhere.